app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -367,6 +367,20 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
         adapter.removeTile(tileId)
     }
 
+    /**
+     * Focus may be lost if all pinned items are removed via onContextItemSelected()
+     */
+    fun checkIfTilesFocusNeedRefresh() {
+        val adapter = tileContainer.adapter as HomeTileAdapter
```

Comment:
Nit: We could be a bit more generic and cast to `RecyclerView.Adapter` instead, since we're only using `itemCount`.

Reply:
Makes sense